spec/controllers/airports_controller_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe Api::V1::AirportsController, type: :controller do
   describe 'GET#index' do
     it 'returns successful response with json-formatted data' do
+      Airport.delete_all
```

Comment:
This should be handled by the DB cleaner. If you are having the issue where your sequencer is not working check out Paige's QQQ from earlier